### PR TITLE
Fix nmcli bug

### PR DESCRIPTION
**Pull request by anivan-suse**
#### Description:

- Change to wireless_disable_interfaces remediation to check for nmcli

### CODE REVIEW

File: linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/bash/shared.sh [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # platform = multi_platform_all
 
-nmcli radio all off
+if 
+    rpm -qa | grep NetworkManager
+then
+    nmcli radio all off
+fi
```
Comment (yuumasato) [inline linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/bash/shared.sh:7]: Could add an `else` clause and print a message stating the reason why the rule was not remediated?

For inspiration:
https://github.com/ComplianceAsCode/content/blob/2f1198047c72452876d7cc12d59204cd479c8b06/linux_os/guide/system/accounts/enable_authselect/bash/shared.sh#L11
https://github.com/ComplianceAsCode/content/blob/2f1198047c72452876d7cc12d59204cd479c8b06/linux_os/guide/system/accounts/enable_authselect/ansible/shared.yml#L29

https://github.com/ComplianceAsCode/content/blob/2f1198047c72452876d7cc12d59204cd479c8b06/linux_os/guide/system/software/integrity/fips/enable_fips_mode/bash/shared.sh#L11

**Comment by marcusburghardt** [comment]
The rule description explicitly mention `nmcli` command and the remediations are based on it. I wonder if would it be better to ensure the `NetworkManager` package (for applicable distros) is installed instead of alerting the user and stopping the remediation. In this case, it sounds reasonable to me to ensure the package installation in the remediation. Any thoughts?

**Comment by marcusburghardt** [comment]
> > The rule description explicitly mention `nmcli` command and the remediations are based on it. I wonder if would it be better to ensure the `NetworkManager` package (for applicable distros) is installed instead of alerting the user and stopping the remediation. In this case, it sounds reasonable to me to ensure the package installation in the remediation. Any thoughts?
> 
> That is a good question, and I'll make more questions to try to answer it. What is the reason that `NetworkManager` is not installed? Wouldn't installing it just to disable a NIC increase the surface attack of the system? Are there other means to disable the wi-fi interface without `nmcli`?
> 
> If `NetworkManager` needs to be installed I'd prefer to have a rule specific for that, so it is more explicit. Also, one could argue this rule should have a CPE platform for NetworkManager package, but I don't feel strongly about it.

I remember to work on this rule some months ago and I deeply researched about alternative ways to disable wireless interfaces. It is possible by manipulating files, but the logic would be "extremely" complex to precisely find the patterns and make the solution stable. My conclusion is that `nmcli` is the best option for this context.

I just remembered the `requires` attribute for the [rules](https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rules). So, in my opinion, this would be the ideal scenario:
- Create a new rule for installing NetworkManager package
   -  We already have the `package_installed` template and this would help.
- Set the `requires` attribute in the `wireless_disable_interfaces` rule, indicating the new rule which installs the NetworkManager package.

What do you think @anivan-suse and @yuumasato ?

**Comment by marcusburghardt** [comment]
> > The rule description explicitly mention `nmcli` command and the remediations are based on it. I wonder if would it be better to ensure the `NetworkManager` package (for applicable distros) is installed instead of alerting the user and stopping the remediation. In this case, it sounds reasonable to me to ensure the package installation in the remediation. Any thoughts?
> 
> @marcusburghardt @yuumasato
> 
> Hey, so I'm pretty confident (not entirely) that we cannot require NetworkManager. Suse supports two networking set-up tools: wicked and networkmanager (which includes nmcli). Wicked is used it graphic-less servers while NetworkManager (nmcli) is used for systems with graphics displays. This is why we went the route of just checking for existence of NetworkManager rather than requiring it.

Thanks for the information @anivan-suse. It is a fair point. In that case I would be fine to only warning the user in this rule.

File: linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/bash/shared.sh [hunk 2]
```diff
@@ -1,3 +1,9 @@
 # platform = multi_platform_all
 
-nmcli radio all off
+if 
+    rpm -qa | grep NetworkManager
```
Comment (yuumasato) [inline linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/bash/shared.sh:4]: I'm sorry I didn't catch this earlier, could you change this to `rpm -q NetworkManager` ?
It is faster and will match the exact package.

**Review by yuumasato — CHANGES_REQUESTED** [review]
The Automatus failures are fine, the rule doesn't have tests and cannot actually be tested in containers.

**Review by yuumasato — APPROVED** [review]

**Pull request merged**